src/components/VideoPlayer/index.js

```diff
@@ -21,8 +21,6 @@ const FILL_NONE = null
 
 const CC_HIDDEN = 'hidden'
 
-const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent)
```

Comment:
removing this as we shouldn't be doing any checking.  if you want the video to work on safari (mobile or otherwise) mute the dang thing and make it part of the experience